src/registry/PartialRegistry.ts

```diff
@@ -12,21 +13,24 @@ const PARTIAL_URI_PLACEHOLDER = '__partial_registry__';
  * Useful for merging with other registries force overrides of subsets of data.
  */
 export interface PartialRegistryOptions {
-  chainMetadata?: ChainMap<Partial<ChainMetadata>>;
-  chainAddresses?: ChainMap<Partial<ChainAddresses>>;
+  chainMetadata?: ChainMap<RecursivePartial<ChainMetadata>>;
+  chainAddresses?: ChainMap<RecursivePartial<ChainAddresses>>;
+  warpRoutes?: Array<RecursivePartial<WarpCoreConfig>>;
```

Comment:
no way we can just make this type `Partial<RegistryOptions>`?

Reply:
It could maybe be a `RecursivePartial` of these but does it matter?